packages/dids/tests/methods/did-dht.spec.ts

```diff
@@ -1214,6 +1228,30 @@ describe('Official DID:DHT Vector tests', () => {
 
     expect(didResolutionResult.didDocument).to.deep.equal(inputDidDocument);
   });
+
+  it('vector 3', async () => {
```

Comment:
yea very shortly we will get these things into the official test vectors package, but this is great for now